httpclient/4.3.1.wso2v2/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~  WSO2 Inc. licenses this file to you under the Apache License,
+  ~  Version 2.0 (the "License"); you may not use this file except
+  ~  in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
```

Comment:
Well, Orbit Bundle Release Guide [1] suggests to have the group id as fixed
in this PR.

[1]
https://docs.google.com/document/d/1I3nWPnG6139YobZzQWPFOUxYEmHxqf9ieWykmQupPtc/edit#heading=h.hoqaqa3z8vuy

On Wed, May 13, 2015 at 4:00 PM, Harsha Kumara <notifications@github.com>
wrote:

> In httpclient/4.3.1.wso2v2/pom.xml
> <https://github.com/wso2/orbit/pull/88#discussion_r30218910>:
>
> > +  ~    http://www.apache.org/licenses/LICENSE-2.0
> > +  ~
> > +  ~ Unless required by applicable law or agreed to in writing,
> > +  ~ software distributed under the License is distributed on an
> > +  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
> > +  ~ KIND, either express or implied.  See the License for the
> > +  ~ specific language governing permissions and limitations
> > +  ~ under the License.
> > +  -->
> > +
> > +<project xmlns="http://maven.apache.org/POM/4.0.0"
> > +         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
> > +         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
> > +
> > +    <modelVersion>4.0.0</modelVersion>
> > +    <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
>
> @manoj-kumara <https://github.com/manoj-kumara> @malithie
> <https://github.com/malithie> I'm having some concern about the group id.
> How about having consistency by changing
> to have group id like org.wso2.apache.httpcomponents or
> org.apache.httpcomponents.wso2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wso2/orbit/pull/88/files#r30218910>.
>



-- 

*Malithi Edirisinghe*
Senior Software Engineer
WSO2 Inc.

Mobile : +94 (0) 718176807
malithim@wso2.com